app/services/you_tube_service.rb

```diff
@@ -5,6 +5,18 @@ def video_info(id)
     get_json('youtube/v3/videos', params)
   end
 
+  def playlist_info(playlist_id)
+    params = { part: 'snippet', id: playlist_id }
+
+    get_json('youtube/v3/playlists', params)
+  end
+
+  def playlist_items(playlist_id)
+    params = { part: 'snippet', maxResults: 50, playlistId: playlist_id }
+
+    get_json('youtube/v3/playlistItems', params)
+  end
```

Comment:
We'll need to expand on this method at some point in order to make additional API calls for paginated results (results > 50). Probably another place we can use `return unless ...`!